doc/modules/ROOT/pages/tutorials/tutorials.adoc

```diff
@@ -1,16 +0,0 @@
-= Tutorials
```

Comment:
this page was likely referenced. We should have a redirect .. but to where?

Reply:
I've brought back the page with a change to avoid the repetition of the list of tutorials.